typings/vscode-mssql.d.ts

```diff
@@ -1155,12 +1163,149 @@ declare module 'vscode-mssql' {
 		createStreamingJobTsql: string;
 	}
 
+	export const enum SchemaCompareEndpointType {
+		Database = 0,
+		Dacpac = 1,
+		Project = 2,
+		// must be kept in-sync with SchemaCompareEndpointType in SQL Tools Service
+		// located at \src\Microsoft.SqlTools.ServiceLayer\SchemaCompare\Contracts\SchemaCompareRequest.cs
```

Comment:
Can you also update the respective comment in STS to include this as a place to be kept in sync?

Reply:
Yes

Reply:
@Benjin, the PR for the respective STS comment: https://github.com/microsoft/sqltoolsservice/pull/2441